ts/parts-more/RadialAxis.ts

```diff
@@ -773,17 +927,27 @@ addEvent(Axis as any, 'init', function (
 
     } else if (polar) {
         extend(this, radialAxisMixin);
-        isCircular = isX;
-        this.defaultRadialOptions = isX ?
-            this.defaultRadialXOptions :
-            merge(this.defaultYAxisOptions, this.defaultRadialYOptions);
 
+        // Check which axis is circular
+        isCircular = this.horiz;
+
+        this.defaultRadialOptions = isCircular ?
```

Comment:
This property should probably be called `this.defaultPolarOptions` since it can hold either radial or circular axis options.

Reply:
Corrected, thanks!